components/Workouts/WorkoutsList/index.jsx

```diff
@@ -0,0 +1,22 @@
+import Heading from "@/components/Atoms/Text/Heading";
+import List from "@/components/List";
+import useSWR from "swr";
+
+export default function WorkoutsList() {
+  const fetcher = (url) => fetch(url).then((res) => res.json());
+  const {
+    data: workouts,
+    isLoading,
+    error,
+  } = useSWR(`/api/workouts/`, fetcher);
+  if (isLoading) return <h2>loading</h2>;
+  if (error) return <h2>error</h2>;
+  if (!workouts) return <h2>Empty database</h2>;
+
+  return (
+    <>
+      <Heading>Workouts</Heading>
+      <List objectList={workouts} />
```

Comment:
```suggestion
      <List workouts={workouts} />
```
`objectList` is a very generic and vague prop name. It says nothing about the data’s purpose, which makes the component harder to understand, reuse or debug. Since you're passing in `workouts` name the prop accordingly